devices/A4988/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Iot.Device.A4988")]
+[assembly: AssemblyCompany("nanoFramework Contributors")]
+[assembly: AssemblyCopyright("Copyright(c).NET Foundation and Contributors")]
+
+[assembly: ComVisible(false)]
+
+/////////////////////////////////////////////////////////////////
```

Comment:
you can remove all the next lines, there are not necessary in this case.

Reply:
fixed